aws/rhel-10.0-nightly-x86_64/config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "user": "ec2-user",
+    "runnerArch": "amd64",
+    "prepareScript": "sudo rm -f /etc/yum.repos.d/*.repo; sudo tee /etc/yum.repos.d/rhel9internal.repo <<EOT\n[RHEL-9-NIGHTLY-BaseOS]\nname=baseos\nbaseurl=https://download.eng.brq.redhat.com/rhel-10/nightly/RHEL-10-Public-Beta/latest-RHEL-10/compose/BaseOS/x86_64/os/\nenabled=1\ngpgcheck=0\n\n[RHEL-9-NIGHTLY-AppStream]\nname=appstream\nbaseurl=https://download.eng.brq.redhat.com/rhel-10/nightly/RHEL-10-Public-Beta/latest-RHEL-10/compose/AppStream/x86_64/os/\nenabled=1\ngpgcheck=0\n\n[RHEL-9-NIGHTLY-CRB]\nname=crb\nbaseurl=https://download.eng.brq.redhat.com/rhel-10/nightly/RHEL-10-Public-Beta/latest-RHEL-10/compose/CRB/x86_64/os/\nenabled=1\ngpgcheck=0\nEOT"
```

Comment:
Urls look fine here, just the repo file name could be updated to something like `rhel10internal.repo`

Reply:
And the name in the `[]` should be updated as well.